developer-workflow/lang-changes.rst

```diff
@@ -23,7 +23,7 @@ Because of the seriousness that language changes carry, any change must be
 beneficial to a large proportion of Python users. If the change only benefits a
 small percentage of Python developers then the change will not be made. A good
 way to see if your idea would work for a large portion of the Python community
-is to ask on :ref:`python-list or python-ideas <communication>`. You can also
+is to ask in the `Ideas category`_ on the `Python Discourse`_. You can also
```

Comment:
This sounds a bit weird to me -- why not just saying ``the `Ideas category on Discourse`_.``?
The second link doesn't seem particularly useful, and it's probably better avoiding two links side by side.

The same applies to several other instances below.

Reply:
Besides re-using existing link targets and being consistent between usages (which are not very convincing reasons), IIRC the main reason for this was to link the internal ref to the section explaining Discourse, but in the end I didn't end up doing that in most cases. In any case, I've revised the various instances to follow the form you suggest here.